gap/main/setup.gi

```diff
@@ -94,6 +103,13 @@ function(x)
   return NrMovedPoints(x![2]) + 1;
 end);
 
+# TODO correct?
```

Comment:
I've no idea if this is correct, but I feel like the TODO should be resolved before merging.